src/scss/components/c-language-switcher.scss

```diff
@@ -3,7 +3,8 @@
     -webkit-appearance: none;
        -moz-appearance: none;
             appearance: none;
-    padding-right: 25px;
+    padding-right: 5px;
```

Comment:
These changes break the language switcher in the footer. What did you try to do?

<img width="158" alt="Language selector with the text overlapping the arrow" src="https://user-images.githubusercontent.com/6073968/37090195-49f9c888-21fb-11e8-9f9e-610c2f73e38b.png">

Reply:
please ignore, I just wanted to upload the main menu